fe1-web/store/reducers/__tests__/SocialReducer.test.ts

```diff
@@ -116,66 +165,312 @@ const filledState4 = {
+    expect(socialReduce(undefined, {} as AnyAction))
+      .toEqual(emptyState);
+  });
+
+  test('reducer should add the first chirp correctly', () => {
+    expect(socialReduce(emptyState, addChirp(mockLaoId, chirp1)))
+      .toEqual(chirpFilledState1);
+  });
 
-test('selector should return the first chirp state', () => {
-  socialReduce(emptyState, addChirp(mockLaoId, chirp1));
-  expect(makeChirpsList().resultFunc(filledState1, mockLaoId))
-    .toEqual([chirp1]);
+  test('reducer should add the newer chirp before the first chirp', () => {
+    expect(socialReduce(chirpFilledState1, addChirp(mockLaoId, chirp2)))
+      .toEqual(chirpFilledState2);
+  });
+
+  test('reducer should add the newer chirp after the second chirp', () => {
+    expect(socialReduce(chirpFilledState2, addChirp(mockLaoId, chirp3)))
+      .toEqual(chirpFilledState3);
+  });
+
+  test('reducer should add the newest chirp on top', () => {
+    expect(socialReduce(chirpFilledState3, addChirp(mockLaoId, chirp4)))
+      .toEqual(chirpFilledState4);
+  });
 });
 
-test('selector should return the newer chirp before the first chirp', () => {
-  socialReduce(filledState1, addChirp(mockLaoId, chirp2));
-  expect(makeChirpsList().resultFunc(filledState2, mockLaoId))
-    .toEqual([chirp2, chirp1]);
+describe('reaction reducer', () => {
+  test('reducer should create entry for a chirp when receiving the first reaction on it', () => {
+    expect(socialReduce(emptyState, addReaction(mockLaoId, reaction1)))
+      .toEqual(reactionFilledState1);
+  });
+
+  test('reducer should add reaction codepoint to an existing chirp', () => {
+    expect(socialReduce(reactionFilledState1, addReaction(mockLaoId, reaction2)))
+      .toEqual(reactionFilledState2);
+  });
+
+  test('reducer should add new reaction sender for a chirp', () => {
+    expect(socialReduce(reactionFilledState2, addReaction(mockLaoId, reaction3)))
+      .toEqual(reactionFilledState3);
+  });
+
+  test('reducer should not add existing sender of a reaction for a chirp', () => {
+    expect(socialReduce(reactionFilledState3, addReaction(mockLaoId, reaction1)))
+      .toEqual(reactionFilledState3);
+  });
+
+  test('reducer should create new chirp entry correctly', () => {
+    expect(socialReduce(reactionFilledState1, addReaction(mockLaoId, reaction4)))
+      .toEqual(reactionFilledState4);
+  });
 });
 
-test('selector should add the newer chirp after the second chirp', () => {
-  socialReduce(filledState2, addChirp(mockLaoId, chirp3));
-  expect(makeChirpsList().resultFunc(filledState3, mockLaoId))
-    .toEqual([chirp2, chirp3, chirp1]);
+describe('chirp selector', () => {
+  test('selector should return an empty list of chirpState when no lao is opened', () => {
+    expect(makeChirpsList().resultFunc(emptyState, undefined))
+      .toEqual([]);
+  });
+
+  test('selector should return an empty list', () => {
+    socialReduce(undefined, {} as AnyAction);
```

Comment:
In this and all the following tests, what is the call to `socialReduce` being used for ? 
My understanding is that we're not saving its return value and not using it in subsequent calls 🤔